src/common/hooks/use-user-session.tsx

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 import { UserSession, AppConfig } from 'blockstack';
+import { UserData } from 'blockstack/lib/auth/authApp';
 
 export const useUserSession = () => {
-  const [userData, setUserData] = React.useState({ loaded: false });
+  const [userData, setUserData] = React.useState<{ loaded: boolean } & Partial<UserData>>({
```

Comment:
Now this creates a type union, would recommend creating a new type.

Reply:
This is updated/changed in a different branch, will avoid changing here.